src/test/java/com/unstoppabledomains/resolution/ResolutionTest.java

```diff
@@ -35,6 +35,30 @@ public static void init() {
         resolution = new Resolution();
     }
 
+    @Test
+    public void testDifferentNetworks() throws Exception {
+        DomainResolution customCnsNetwork = Resolution.builder()
+            .providerUrl(NamingServiceType.CNS, "https://rinkeby.infura.io/v3/e0c0cb9d12c440a29379df066de587e6")
+            .providerUrl(NamingServiceType.ENS, "https://goerli-light.eth.linkpool.io/")
+            .chainId(NamingServiceType.ZNS, Network.KOVAN)
+            .build();
+            
+        Network customCnsChainId = customCnsNetwork.getChainId(NamingServiceType.CNS);
+        Network customEnsChainId = customCnsNetwork.getChainId(NamingServiceType.ENS);
+        Network customZnsChainId = customCnsNetwork.getChainId(NamingServiceType.ZNS);
+        assertEquals(Network.RINKEBY, customCnsChainId);
+        assertEquals(Network.GOERLI, customEnsChainId);
+        assertEquals(Network.KOVAN, customZnsChainId);
+
+        DomainResolution defaultSettings = new Resolution();
```

Comment:
@JohnnyJumper Please, split the the test into two: one for default networks and second for custom.\

Use Infura endpoints in tests.